app/templates/twoDiv.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'layout.html' %}
+
+{% block content %}
+<div class="two-div-content row">
+    <!-- Logan Kiser: "col-md-8" will assign 2/3 of the content container to the
+                 left div, since the right div is "col-md-4" ** 2/3 = 8 / (8 + 4) ** -->
+    <div class="leftcontent col-md-8">{% block leftblock %}{% endblock %}</div>
+    <div class="rightcontent col-md-4">{% block rightblock %}{% endblock %}</div>
+</div>
+{% endblock %}
```

Comment:
Here's that intermediary layout/structure template that a dev could reach for if they choose to. Restricting layout to the use of one of these templates would reduce work (should only need to do this once for all pages that choose this structure) and increase consistency between pages (we aren't accidentally styling things differently between pages using the same structure).

Reply:
twoDiv would extend layout because it bolts into the block content.  We could then dynamically page load lefblock and rightblock depending on url path

Reply:
though I want to clarify that we will likely never have to invoke twoDiv.html from a route.  The jinja templates will extend twoDiv, which extends layout.html.  So calling leftblock and right block jinja based on page URL will automatically pull twodive and layout.

Reply:
Absolutely!